services/app/src/actions/Card.tsx

```diff
@@ -17,6 +17,11 @@ export interface ToCardArgs {
 export const toCard = remoteify(function toCard(a: ToCardArgs, dispatch: Redux.Dispatch<any>, getState?: () => AppStateWithHistory): ToCardArgs {
   const nav = getNavigator();
   const state = (getState !== undefined) ? getState() : getStore().getState();
+  // Disable clicking on the same menu again
```

Comment:
Messing with the toCard action is pretty dangerous since it affects the functionality of the entire app. What problem were you seeing that made you want to add this?

Reply:
So before this behaviour was like this https://www.useloom.com/share/ac0e84d5fc194eaf98aab1e34819e6cd

after the change
https://www.useloom.com/share/3e4102b97ef64e65946019ccf12de9fd

Reply:
Gotcha. An even better solution here would be to disable the menu item for the page the user is currently on - not only would that prevent this issue, it would also give the user a visual indication of which menu option / page they're on

Reply:
I will create an issue for this behaviour, as this feature is beyond the scope of this PR.

Reply:
Sounds good. In the meantime let's revert this change to the to card action. Thanks!

Reply:
Yeah I have already reverted it